genet/cli.py

```diff
@@ -1,153 +1,12 @@
-import json
 import logging
-import os
-import time
 from pathlib import Path
 from typing import Any, Optional
-
 import click
-import geopandas as gpd
-import pandas as pd
-from pyproj import CRS
-
-import genet
-import genet.output.sanitiser as sanitiser
-import genet.utils.spatial as spatial
-from genet import (
-    google_directions,
-    read_gtfs,
-    read_matsim,
-    read_matsim_schedule,
-    read_osm,
-)
-from genet.core import Network
-from genet.output.geojson import (
-    generate_headway_geojson,
-    generate_speed_geojson,
-    modal_subset,
-    save_geodataframe,
-)
-from genet.utils.persistence import ensure_dir
-from genet.variables import EPSG4326
 
 logging.basicConfig(level=logging.INFO, format="%(levelname)-3s %(message)s")
```

Comment:
Should the logging config live "deeper" inside the library? Would we want it to be consistently configured no matter what route the user is taking (CLI, web app, Python script, notebook, etc.) into the code?